### PR TITLE
Meeting Schedule & Notes Page

### PR DESCRIPTION
**Pull request by kmpaul**
Creating a draft PR to address #8.  We need to finalize a meeting schedule (day and time) and then I will:

- [x] Create a Google Calendar for these recurring meeting times.
- [x] Inform Ryan Abernathey of these times so they can be added to the Pangeo Calendar, too.
- [x] Update the "Meeting Schedule & Notes" page with the correct times and embedded Google Calendar.

### CODE REVIEW

**Comment by kmpaul** [comment]
I don't think that we need a special Google Calendar.  I've informed Ryan Abernathey, and he has offered to add these meeting times to the Pangeo Google Calendar, which you can get the link to here:

https://calendar.google.com/calendar/embed?src=ucar.edu_c23ln4014khs3f65o93vqv5kqc%40group.calendar.google.com 

I think that is sufficient.

I'll work on adding the meeting times to the Pythia Portal.

**Comment by kmpaul** [comment]
I made a Google Calendar anyway so that we can link it to the Meetings page on the Portal:

<iframe src="https://calendar.google.com/calendar/embed?src=c_4qpvf316afd9mv0ci7d2uiafog%40group.calendar.google.com" style="border: 0" width="800" height="600" frameborder="0" scrolling="no"></iframe>

**Comment by kmpaul** [comment]
I think the only think I am waiting on before opening this up to review is a Zoom meeting time.

@jukent:  Do you might using your NCAR Zoom account to create a recurring Zoom meeting at 11AM Mountain every Thursday with the following settings (thanks to Ryan Abernathey)?


image

**Comment by clyne** [comment]
> I made a Google Calendar anyway so that we can link it to the Meetings page on the Portal:
> 
> https://calendar.google.com/calendar/u/0?cid=Y180cXB2ZjMxNmFmZDltdjBjaTdkMnVpYWZvZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t

Is it worth the bother of having and maintaining a separate calendar for two meetings? Maybe we just list the time and coordinates on the website and have one less thing to keep track of... Just a thought.

**Comment by kmpaul** [comment]
It is definitely debatable whether having a separate calendar is useful, but it is nice to allow other people to easily add our meeting to *their* calendar.  With a GCAL, they can add the Pythia calendar (with all of our meetings) to their own calendar with a single click.  And if we change our meeting times, then they get updated.

If the only people we had to worry about communicating with was our own internal team, then I agree that having the separate calendar is not very useful.  But is is very useful for communicating with people external to our organizations.

At least those are my thoughts.  Do others feel differently?

**Comment by jukent** [comment]
> I think the only think I am waiting on before opening this up to review is a Zoom meeting time.
> 
> @jukent: Do you might using your NCAR Zoom account to create a recurring Zoom meeting at 11AM Mountain every Thursday with the following settings (thanks to Ryan Abernathey)?
> 
> image

I was able to create a Zoom meeting (the meeting ID is different -- but consistent for each meeting -- since I do not know how to edit that). 

Could you give me editing powers over the Pythia calendar so that I may add the Zoom meetings to that calendar?

**Comment by kmpaul** [comment]
@jukent:  I just added you as a maintainer for the Pythia Google Calendar.

**Comment by kmpaul** [comment]
And I think 1 recurring weekly Zoom meeting should be fine.

**Comment by jukent** [comment]
Hi everyone. For some reason the functionality to add a Zoom meeting to a Google calendar doesn't seem to be working right now. I'm going to post the invitation here (since it is public anyway) so that we have the information before our 11:30 today. I am confident I can figure it out later today.

--------------
Julia Kent is inviting you to a scheduled Zoom meeting.

Topic: Project Pythia Meeting
Time: Jan 7, 2021 11:30 AM Mountain Time (US and Canada)
        Jan 7, 2021 11:30 AM
        Jan 14, 2021 11:00 AM
        Jan 21, 2021 11:00 AM
        Jan 28, 2021 11:00 AM
        Feb 4, 2021 11:00 AM
        Feb 11, 2021 11:00 AM
        Feb 18, 2021 11:00 AM
        Feb 25, 2021 11:00 AM
        Mar 4, 2021 11:00 AM
        Mar 11, 2021 11:00 AM
        Mar 18, 2021 11:00 AM
        Mar 25, 2021 11:00 AM
        Apr 1, 2021 11:00 AM
        Apr 8, 2021 11:00 AM
        Apr 15, 2021 11:00 AM
        Apr 22, 2021 11:00 AM
        Apr 29, 2021 11:00 AM
        May 6, 2021 11:00 AM
        May 13, 2021 11:00 AM
        May 20, 2021 11:00 AM
Please download and import the following iCalendar (.ics) files to your calendar system.
Weekly: https://ucar-edu.zoom.us/meeting/tJUucOyspjwvEtO2Vsv3nFczYnNAQx2aNCGF/ics?icsToken=98tyKuCprTwvHtyStRGARowcA4_CZ-jxiGZYj7dyzin2FXNdRADsD-xsE4NnQtn-

Join Zoom Meeting
https://ucar-edu.zoom.us/j/91375487587

Meeting ID: 913 7548 7587
Passcode: ugtmQ34M
One tap mobile
+16699006833,,91375487587#,,,,*24478313# US (San Jose)
+12532158782,,91375487587#,,,,*24478313# US (Tacoma)

Dial by your location
        +1 669 900 6833 US (San Jose)
        +1 253 215 8782 US (Tacoma)
        +1 346 248 7799 US (Houston)
        +1 929 205 6099 US (New York)
        +1 301 715 8592 US (Washington D.C)
        +1 312 626 6799 US (Chicago)
Meeting ID: 913 7548 7587
Passcode: 24478313
Find your local number: https://ucar-edu.zoom.us/u/aHS56gsAZ

**Comment by jukent** [comment]
The Zoom meeting information has been added to the calendar events.

**Comment by kmpaul** [comment]
@jukent:  It looks like the Zoom meeting has a password, which isn't supposed to be set.  Can you confirm that the meeting was set up with the same settings as shown above?

**Comment by jukent** [comment]
> @jukent: It looks like the Zoom meeting has a password, which isn't supposed to be set. Can you confirm that the meeting was set up with the same settings as shown above?

Hmm for some reason it won't let me not have a password on recurring meetings. I will go ahead and fix this for the upcoming meeting ASAP and then try to figure this out.

**Comment by kmpaul** [comment]
Ok.  It looks like we can no longer create a recurring Zoom meeting without a password, at least the way NCAR's Zoom account is set up.  So, we'll just proceed with a password.

**Comment by kmpaul** [comment]
@clyne, @jukent:  I think this PR is now ready for review.

**Comment by clyne** [comment]
@kmpaul is there a way to build the documentation and preview it locally?

**Review by dopplershift — APPROVED** [review]

**Comment by brian-rose** [comment]
> @kmpaul is there a way to build the documentation and preview it locally?

@clyne yes! From a local clone of the source repository, do this:
```
conda env create -f ci/environment.yml
conda activate
make html
```
This will build the site in a dedicated conda environment with all the same packages as are used to build it here on GitHub.

I'll try to submit a separate PR that describes how to do this briefly on the Contributors page.

**Comment by brian-rose** [comment]
Though I imagine there's a way to automate this? Can we have the site rendered and available as a zip file download anytime a new PR is pushed? That would be very useful.

**Comment by kmpaul** [comment]
Thanks, @brian-rose!  Yes, @clyne, that's how you do it.  Sorry.  I'm in another meeting.

Following @brian-rose's instructions, then just do (on a Mac):

```text
$ open _build/html/index.html
```

**Comment by kmpaul** [comment]
Oh, and these steps have to be done from the `content` directory.

**Comment by kmpaul** [comment]
I think these instructions should be in the `README.md` *and* in the `CONTRIBUTING.md` page.

**Comment by brian-rose** [comment]
@kmpaul I'll whip something up.

**Comment by clyne** [comment]
> > @kmpaul is there a way to build the documentation and preview it locally?
> 
> @clyne yes! From a local clone of the source repository, do this:
> 
> ```
> conda env create -f ci/environment.yml
> conda activate
> make html
> ```
> 
> This will build the site in a dedicated conda environment with all the same packages as are used to build it here on GitHub.
> 
> I'll try to submit a separate PR that describes how to do this briefly on the Contributors page.

I had to install sphinx, and now I'm getting:

(base) bash-3.2$ make html
Running Sphinx v3.2.1

Extension error:
Could not import extension myst_nb (exception: No module named 'myst_nb')
make: *** [html] Error 2

**Comment by brian-rose** [comment]
> Ok. It looks like we can no longer create a recurring Zoom meeting without a password, at least the way NCAR's Zoom account is set up. So, we'll just proceed with a password.

How do we plan to distribute the password? Currently the Meetings page provides the zoom link but doesn't say anything about the passcode.

**Comment by brian-rose** [comment]
> > > @kmpaul is there a way to build the documentation and preview it locally?
> > 
> > 
> > @clyne yes! From a local clone of the source repository, do this:
> > ```
> > conda env create -f ci/environment.yml
> > conda activate
> > make html
> > ```
> > 
> > 
> > This will build the site in a dedicated conda environment with all the same packages as are used to build it here on GitHub.
> > I'll try to submit a separate PR that describes how to do this briefly on the Contributors page.
> 
> I had to install sphinx, and now I'm getting:
> 
> (base) bash-3.2$ make html
> Running Sphinx v3.2.1
> 
> Extension error:
> Could not import extension myst_nb (exception: No module named 'myst_nb')
> make: *** [html] Error 2

@clyne looks like you are not running `make html` from the correct conda environment.

I think I mis-typed above, you should do this:
```
conda env create -f ci/environment.yml
conda activate pythia
make html
```
All the dependencies (including sphinx and extensions) should then be in the current conda environment.

**Comment by clyne** [comment]
that did it. Thanks!

**Comment by clyne** [comment]
> > Ok. It looks like we can no longer create a recurring Zoom meeting without a password, at least the way NCAR's Zoom account is set up. So, we'll just proceed with a password.
> 
> How do we plan to distribute the password? Currently the Meetings page provides the zoom link but doesn't say anything about the passcode.

@brian-rose with the exception of today's meeting I'm seeing the password in the calendar entries...

**Review by clyne — APPROVED** [review]
Looks good!

**Comment by brian-rose** [comment]
> > > Ok. It looks like we can no longer create a recurring Zoom meeting without a password, at least the way NCAR's Zoom account is set up. So, we'll just proceed with a password.
> > 
> > 
> > How do we plan to distribute the password? Currently the Meetings page provides the zoom link but doesn't say anything about the passcode.
> 
> @brian-rose with the exception of today's meeting I'm seeing the password in the calendar entries...

True! @kmpaul maybe add a note on the page mentioning that the passcode can be found in the calendar entries. If you just click on the link as given in the first sentence, zoom prompts for a passcode.

**Comment by kmpaul** [comment]
@brian-rose:  Would it be better to just remove the link to the Zoom meeting, and instead point people to the Google Calendar for information about how to join?

**Comment by brian-rose** [comment]
Agreed it would be great to get rid of the passcode, but institutional rules are what they are. It would be the same if we used a UAlbany-hosted zoom meeting.

I guess I would vote for removing the first link altogether and just saying something like "We will meet on zoom, please find links in the calendar below".

File: content/pages/meetings.md
Comment (kmpaul) [inline content/pages/meetings.md:5]:
```suggestion
Project Pythia holds weekly public meetings every Thursday at 11:00AM Mountain (1:00PM Eastern) on Zoom.  Please find links and information about how to join in the calendar below.
```

**Review by kmpaul — COMMENTED** [review]
Implementing @brian-rose's suggestion.

**Comment by kmpaul** [comment]
Is this ready to merge?

**Review by andersy005 — APPROVED** [review]

**Review by brian-rose — APPROVED** [review]

**Comment by brian-rose** [comment]
Looks great to me.

**Pull request merged**
